README.md

```diff
@@ -1384,6 +1386,56 @@ out  | evt.ota_end.report      | object     | Sent on upgrade end with upgrade s
    }
 }
 ```
+### Virtual meter service
+
+This service enables a device to report accumulated consumption in case it does not have its own metering capabilities (eg. a relay with output binary switch service or thermostat). Thanks to the service accumulated consumption is calculated based on manually provided consumption in a unit of time (eg. watts, m3/h). A device shall calculate cumulated consumption at every state's change, mode change and at least every interval value - 30 minutes by default. When this service is added on a device, appropriate metering service is created (eg. meter_elec, meter_water) and a device starts to send measurements.
```

Comment:
Thanks to `this` service, accumulated consumption is...

`The service` shall calculate...

and `the service` starts to send measurements

Reply:
1st I think is wrong?
2nd - ok
3rd - ok

Reply:
Unless requirements changed this service should be capable of spoofing both pulse and cumulative measurements, so in the case of electric energy mobile app in my opinion should deliver:

```
{
   "type": "cmd.meter.add",
   "serv": "meter_virtual",
   "val_t": "float_map",
   "val": {
      "off": 1.5,
      "on": 500,
   },
   "props": {
      "unit": "W"
   }
}
```

But in the result added meter_elec service should be both supporting W and kWh.

Reply:
You probably forgot about sth, json you pasted is identical with my propasal

Reply:
Let's have a call.